sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -57,6 +57,27 @@ def get_observation_log_response(*args, **kwargs):
         )
 
 
+def create_namespaced_persistent_volume_claim_response(*args, **kwargs):
+    if kwargs.get("namespace") == "pvc creation failed":
```

Comment:
Please can you create constant for namespace that you want to use to check the PVC creation error ?
Like what we do in Training Operator: https://github.com/kubeflow/training-operator/blob/master/sdk/python/kubeflow/training/api/training_client_test.py#L29

Reply:
Sure, I have created a constant for that.